app/views/export/index.html.slim

```diff
@@ -97,6 +100,11 @@ p
   =link_to('here', 'https://github.com/benwbrum/fromthepage/wiki/FromThePage-Support-for-the-IIIF-Presentation-API-and-Web-Annotations')
   '.
 
+p
+  'Collection API Endpoint:
```

Comment:
Let's change the phrasing to: `IIIF Collection API Endpoint:` and then list the URI itself, linked to `iiif_collection_path(@collection.id)`
So:
IIIF Collection API Endpoint: [http://localhost:3000/iiif/collection/2](http://localhost:3000/iiif/collection/2)